dotnet-core-music-linux/azuredeploy.json

```diff
@@ -0,0 +1,435 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "vmName": {
```

Comment:
@bmoore-msft for the notes here. For this one specifically, what are you recommending I move to a variable?